### ISSUE TITLE
Editor crash on single prompt preview

### ISSUE THREAD

**Issue by dkoo** [Automattic/newspack-popups#556]
This may be an error with the Newspack Components NPM package, but I couldn't reproduce in the components demo page nor from the Campaigns dashboard (also in Newspack Plugin), only here. 

1. Edit a prompt
2. Preview the prompt from the block editor
3. Observe a hard editor crash and JS console error:

<img width="946" alt="Screen Shot 2021-06-09 at 11 21 12 AM" src="https://user-images.githubusercontent.com/2230142/121400449-d4b39580-c914-11eb-8319-92beeda4310a.png">

```
_icons is undefined
    WebPreview webpack:///./node_modules/newspack-components/dist/cjs/web-preview/index.js?:99
    render webpack:///./node_modules/newspack-components/dist/cjs/web-preview/index.js?:229
    Ie https://siteurl/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.13.1:104
    [...]
```

Seems to stem from not being able to import the `@wordpress/icons` package, but the latest version of this package is properly defined as a dependency in Newspack Components, so I'm not sure what the problem is.

**Comment by dkoo** [comment]
I spent a lot of time trying to debug this, without getting anywhere so far. Seems to be a pretty hairy dependency bug related to the replacement of Material icons with WP icons in #982. The only thing I could find is that when following the links from the JS error into the dist file for `web-preview.js`, webpack seems to be calling `@wordpress/icons` using the wrong path. Note the paths for `i18n` and `element` vs. `icons`:

```
var _element = __webpack_require__(/*! @wordpress/element */ "@wordpress/element");

var _i18n = __webpack_require__(/*! @wordpress/i18n */ "@wordpress/i18n");

var _icons = __webpack_require__(/*! @wordpress/icons */ "./node_modules/@wordpress/icons/build-module/index.js");
```

However, I'm not sure what this means and I couldn't figure out how to update it. Things I tried with no luck:

* Installing `@wordpress/icons` as a top-level dependency in Campaigns' package.json
* Running `npm install` using various versions of NPM
* Downgrading webpack to the same version used in `newspack-plugin`

I wonder if this is a bug in `@wordpress/icons` itself that we won't be able to resolve in our code?

**Comment by adekbadek** [comment]
When testing with different versions of `newspack-components`, I've discovered that version `1.3.0` is the last one without this issue. Not surprisingly, [`1.4.0` is when usage of `@wordpress/icons` was introduced](https://diff.intrinsic.com/newspack-components/1.3.0/1.4.0#file-src__web-preview__index.js).

> I wonder if this is a bug in @wordpress/icons itself that we won't be able to resolve in our code?

I think it rather has something to do with the dependency structure between Campaigns and `newspack-components`, because on a fresh project, `@wordpress/icons` can be installed and used without issues. Worth mentioning, `newspack-components`' installation fails on a fresh project (both most recent and `1.3.0` versions).

**Comment by adekbadek** [comment]
Cannot reproduce on current `master`, can you confirm @Automattic/newspack-product ?

**Comment by dkoo** [comment]
Confirmed, this is no longer reproducible with the current `master`. Maybe fixed by #611?
